### DISCUSSION TITLE
Ability to set up with Remote computer

### DISCUSSION THREAD

**Discussion by jimmyleith** [fredemmott/StreamDeck-Discord#81] [Ideas]
Is there a way this can function with Remote workstations, for use for users with dedicated stream machines? Other StreamDeck applications Such as Spotify by BarRaider function this way and can control remotely by API, or is this a limitation?

**Comment by fredemmott** [comment]
Sorry, no, this is up to the way the specific application's API works. It's possible with spotify as that's remote control over the internet (or any network), but discord's local control API only works on the same machine.

**Reply by jimmyleith** [reply]
Thanks for the super fast reply! Can you think of any scuffed workarounds to assist with this? ATM i am resorting to have to use a macropad just to mute discord on the same machine. If not all is well.

**Reply by fredemmott** [reply]
No; the best approach would be if Elgato supported remote streamdeck actions in general (not just the phone apps) rather than needing each plugin to implement it; perhaps reach out to support@elgato.com to let them know it's a feature you'd like? The more people who do that the greater chance they'll do it :)
